addons/sale/wizard/res_config_settings_views.xml

```diff
@@ -63,45 +63,34 @@
-                                    <button name='%(payment.action_payment_provider)d' icon="oi-arrow-right" type="action" string="Payment Providers" class="btn-link"/>
-                                </div>
-                                <div class="mt16" title="Days between quotation proposal and expiration. 0 days means automatic expiration is disabled">
-                                    <label for="quotation_validity_days"/>
-                                    <field name="quotation_validity_days" class="text-center" style="width: 5rem;"/>
-                                    <div class="d-inline-block">days</div>
-                                </div>
+                        <setting id="online_signature" company_dependent="1"
+                                 documentation="https://www.odoo.com/documentation/17.0/applications/sales/sales/send_quotations/get_signature_to_validate.html"
+                                 help="Request customers to sign quotations to validate orders. The default can be changed per order or template.">
+                            <field name="portal_confirmation_sign"/>
+                        </setting>
+                        <setting id="online_payment" company_dependent="1"
+                                 documentation="https://www.odoo.com/documentation/17.0/applications/sales/sales/send_quotations/get_paid_to_validate.html"
+                                 help="Request a payment to confirm orders, in full (100%) or partial. The default can be changed per order or template.">
+                            <field name="portal_confirmation_pay"/>
+                            <div invisible="not portal_confirmation_pay" class="oe_row fw-bold">
+                                Payment
+                                <field name="prepayment_percent" widget="percentage" class="oe_inline"/>
+                            </div>
+                            <div class="mt8" invisible="not portal_confirmation_pay">
+                                <button name='%(payment.action_payment_provider)d' icon="oi-arrow-right" type="action" string="Payment Providers" class="btn-link"/>
+                            </div>
+                        </setting>
+                        <setting id="quotation_validity_days">
+                            <div title="Days between quotation proposal and expiration. 0 days means automatic expiration is disabled">
+                                <label for="quotation_validity_days"/>
+                                <field name="quotation_validity_days" class="text-center" style="width: 3rem;"/>
+                                <div class="d-inline-block">days</div>
+                                <span class="fa fa-lg fa-building-o p-2"
+                                      title="Values set here are company-specific."
+                                      groups="base.group_multi_company"/>
```

Comment:
The company_dependent label was missing for the quotation validity days, I pushed to add this part ;).